sdk/state/update.go

```diff
@@ -0,0 +1,143 @@
+package state
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/stellar/experimental-payment-channels/sdk/txbuild"
+	"github.com/stellar/go/xdr"
+)
+
+type PaymentProposal struct {
```

Comment:
I think you should use the term `Payment` here, rather than `PaymentProposal`. My reasoning for this is that it starts out as a proposal, but once all its payments are adopted, it is a complete payment and is no longer only a proposal.

Reply:
that's a good point, changed to `Payment`
https://github.com/stellar/experimental-payment-channels/pull/55/commits/e2211d4850c38916a7bead62e8d1c2f7370704ab